glossary.yml

```
@@ -2082,7 +2096,13 @@
       "camel case" werden keine Leerzeichen, Unterstriche (`_`), Punkte (`.`)
       oder Bindestriche (`-`) verwandt und jedes Wort großgeschrieben.
       Beispiele sind: `BerechneSumme` oder `findeMuster`.
-
+  tn:
+    term: "Mokwalo wa Kamela"
+    def: >
+      Ke mokgwa wa go kwala khoutu ka go sa tsenye matshwao a sekgala (space), 
+      thalela (_), khutlo (.) kana thaladi (-) mo maineng a di’variable kana a dilo (objects). 
+      Mo godimo ga moo, lefoko lengwe le lengwe mo maineng a le similolola ka tlhakakgolo. 
+      Dikai tsa mokwalo o ke “CalculateSum” le “findPattern”.
```

Style: code examples should be wrapped in backticks. 

```suggestion
      Ke mokgwa wa go kwala khoutu ka go sa tsenye matshwao a sekgala (space), 
      thalela (`_`), khutlo (`.`) kana thaladi (`-`) mo maineng a di’variable kana a dilo (objects). 
      Mo godimo ga moo, lefoko lengwe le lengwe mo maineng a le similolola ka tlhakakgolo. 
      Dikai tsa mokwalo o ke `CalculateSum` le `findPattern`.
```

---

I totally agree with 2104  and 2105. However I do not totally agree with the first two.

2102 :  I would prefer a little word phrase that means code instead of writing khoutu.  Code is a set of instructions. So a Setswana equivalent  for "a set of instructions" will do.

2103: Same sentiment as the above (2102).  What can be a Setswana phrase equivalent to "variable" instead of di'variable 

Wonderful work. I do appreciate that it is hard and our language is quite limited espeacilly in  tech language. Absolutely great work.

---

Hi @MaucoKL and @zkamvar . Sorry for the delayed response. And thank you for your review. I was still figuring out alternative translations for the terms you suggested. So far I've come up with "dikaelo tsa khomputara" for code, but I'm still stuck at variable. I'm doing my research and I should come up with a better term in a few days.

---

Not a problem. I know its very hard...  all the best

---

Hi @MaucoKL. Here is what I have come up with regarding translations for "variables":

1. Difetogi
2. Boleng jo bo fetogang
3. Dipharologano
4. Maemedi a boleng jo bo fetogang/farologanang
5. Maemedi 

I particularly like 2 and 4 as they are very descriptive, but I worry that 4 might be too wordy. I would love to hear your opinion on which term is apt for this translation. Thank you once again for reviewing my contribution.

---

Hi @zkamvar. Do I need to open a new pull request to enact changes suggested by reviewers?

---

4 or 5 sounds good.

---

Thank you. I will go with 4 then. So here is a "preview" of the changes I'll make:

1. Code: Dikaelo tsa khomputara
2. Variables: Maemedi a boleng jo bo fetogang

---

I'm not sure if I should open a new pull request but I will enact the changes soon. I appreciate your feedback. It has pushed me to think harder because, like you mentioned, Setswana does not have a well-developed ICT vocabulary.

---

great.!

---

> I'm not sure if I should open a new pull request

Hello @GorataB, you do not need to open a new pull request. You can continue to make changes to your fork and they will automatically update in this pull request.

---

Hi @zkamvar. Thank you, I just made the change.